actionpack/lib/abstract_controller/base.rb

```diff
@@ -9,6 +9,8 @@
 module AbstractController
   # Raised when a non-existing controller action is triggered.
   class ActionNotFound < StandardError
+    include ActiveSupport::CorrectableError
```

Comment:
How about `include DidYouMean::Correctable` instead like #42379?
We don't want to extract the internal `ActiveSupport::CorrectableError` in Active Support.

Reply:
I'll see if I can make these changes.

Reply:
@kamipo I've changed the MR to use `DidYouMean::Correctable`